scaletest/templates/scaletest-runner/scripts/run.sh

```diff
@@ -13,15 +13,21 @@ log "Running scaletest..."
 set_status Running
 
 start_phase "Creating workspaces"
-coder exp scaletest create-workspaces \
-	--count "${SCALETEST_PARAM_NUM_WORKSPACES}" \
-	--template "${SCALETEST_PARAM_TEMPLATE}" \
-	--concurrency "${SCALETEST_PARAM_CREATE_CONCURRENCY}" \
-	--timeout 5h \
-	--job-timeout 5h \
-	--no-cleanup \
-	--output json:"${SCALETEST_RESULTS_DIR}/create-workspaces.json"
-show_json "${SCALETEST_RESULTS_DIR}/create-workspaces.json"
+if [[ ${SCALETEST_PARAM_SKIP_CREATE_WORKSPACES} == 0 ]]; then
+	# Note that we allow up to 5 failures to bring up the workspace, since
+	# we're creating a lot of workspaces at once and some of them may fail
+	# due to network issues or other transient errors.
+	coder exp scaletest create-workspaces \
+		--retry 5 \
```

Comment:
I'm curious if this must be ignored&retried, maybe we should identify and fix problems in a different way, even provisioning workspaces slower?

Anyway, consider it to be a side thought.

Reply:
We would have to ensure terraform registry is cached locally, once we know workspace builds don't use any external networking, we could potentially disable the retry, if we want. 👍🏻